builtin/logical/pki/path_config_acme.go

```diff
@@ -276,14 +278,15 @@ func (b *backend) pathAcmeWrite(ctx context.Context, req *logical.Request, d *fr
 	return genResponseFromAcmeConfig(config), nil
 }
 
-func isPublicACMEDisabledByEnv() bool {
+func isPublicACMEDisabledByEnv(log hclog.Logger) bool {
 	disableAcmeRaw, ok := os.LookupEnv(disableAcmeEnvVar)
 	if !ok {
 		return false
 	}
 
 	disableAcme, err := strconv.ParseBool(disableAcmeRaw)
 	if err != nil {
+		log.Warn("could not parse env var VAULT_DISABLE_PUBLIC_ACME", "error", err)
```

Comment:
Ah, I was thinking about making this an error here, and then surfacing that error in the `pathAcmeWrite` handler since there we have an (admin-ish) user's attention IMO.

Reply:
Ah I get what your saying now, yeah that does make sense and will update.

Reply:
Updated within https://github.com/hashicorp/vault/pull/20600/commits/7bf799ce0ca14cab793fccae7ff3b5138127f02d